Lib/http/__init__.py

```diff
@@ -26,7 +26,6 @@ class HTTPStatus:
     def __new__(cls, value, phrase, description=''):
         obj = int.__new__(cls, value)
         obj._value_ = value
-
```

Comment:
This whitespace change can probably be omitted?

Reply:
In theory, but the blank line irritates me.  :)

Reply:
Maybe I'm nitpicking, but that means you're making a change to `HTTPStatus` while this PR is not about HTTPStatus but about Enums. It's a whitespace change, but still.. I've made it now part of my PR https://github.com/python/cpython/pull/117611/files which already changes the file `Lib/http/__init__.py` -- so you can drop it from this PR if you want.

Reply:
Fair point.  Change removed.